### PR TITLE
transition to postgres

### PR DESCRIPTION
**Pull request by mwickett**
* Transitions to postgresql rather than sqlite for local dev.
* I've added a docker command to the readme that should get things up and running.
* Adds vercel build commands to run db migrations in staging and prod

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    "superjson-next" // 👈
+  ]
```
Comment (mwickett) [inline .babelrc:5]: This gets around an issue with NextJS `getStaticProps` serializable requirements, and Prisma returns some `Date` objects that aren't. See https://github.com/blitz-js/superjson#using-with-nextjs

File: pages/index.tsx [hunk 2]
```diff
@@ -42,8 +43,11 @@ export default function Home({ feed }) {
 }
 
 export async function getServerSideProps() {
-  const res = await fetch('http://localhost:3000/api/feed')
-  const feed = await res.json()
+  const feed = await prisma.story.findMany({
+    where: { approved: true },
+    orderBy: { createdAt: 'asc' },
+  })
+
```
Comment (mwickett) [inline pages/index.tsx:50]: Get data directly from the DB rather than through the API route.

**Comment by mwickett** [comment]
I'm power merging this - feel free to review retroactively and we can fixup anything you spot.

File: pages/index.tsx (hunk 2 above)
Comment (silent1mezzo) [reply pages/index.tsx:50]: How does the frontend have access to the DB?

**Pull request merged**